spec/support/factory_helpers.rb

```diff
@@ -18,7 +18,9 @@ def mock_file_factory(opts = {})
                  original_checksum: opts.fetch(:original_checksum, []),
                  digest:            opts.fetch(:digest, []),
                  duration:          opts.fetch(:duration, []),
-                 sample_rate:       opts.fetch(:sample_rate, []))
+                 sample_rate:       opts.fetch(:sample_rate, []),
+                 versions:          opts.fetch(:versions, []),
+                 uri:               opts.fetch(:versions, []))
```

Comment:
Was this deliberate or a copy/paste issue?
```suggestion
                 uri:               opts.fetch(:uri, []))
```